modules/rostests/apitests/atl/CString.inl

```diff
@@ -17,75 +17,75 @@ TEST_NAMEX(operators_init)
     CStringX first(_X("First "));
     ok(first.IsEmpty() != true, "Expected first to not be empty\n");
     ok(first.GetLength() == 6, "Expected GetLength() to be 6, was: %i\n", first.GetLength());
-    ok_int(first.GetAllocLength(), 6);
+    //ok_int(first.GetAllocLength(), 6);
```

Comment:
why the disabled tests?

Reply:
The test fails on msvc's ATL, and it is an irrelevant implementation detail anyway.